velox/docs/functions/spark/conversion.rst

```diff
@@ -24,6 +24,23 @@ Valid examples:
   SELECT cast(1234 as tinyint); -- -46
   SELECT cast(1234567 as smallint); -- -10617
 
+From interval year month
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Since Presto also uses this implementation, we may also document the Presto behaviour in https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/presto/conversion.rst.